env/Tensorflow_probability.yml

```diff
@@ -0,0 +1,33 @@
+name: Tensorflow
+channels:
+  - conda-main
+  - conda-mo-internal
+  - conda-forge
+  - conda-r
+dependencies:
+  - python=3.7
+  - tensorflow-probability=0.12
+  - ipykernel
+  - tensorboard=2.4
+  - notebook
+  - matplotlib
+  - keras-applications
+  - pydot
+  - graphviz
+  - cf_units
+  - cf-units=3.0.0
+  - iris=3.1.0
+  - dask
+  - seaborn
+  - scikit-learn
+  - jupyterlab
+  - xarray=0.18.2
+  - ipympl
+  - ipywidgets
+  - nodejs
+  - numpy==1.19
+  - line_profiler
+  - gpflow
+  - pysr
+  - tensorflow-datasets
+prefix: /home/h06/ncreaser/.conda/envs/Tensorflow
```

Comment:
this should be removed manually from this file.

Reply:
ooops